dist/22-5490-schema.json

```diff
@@ -25,7 +25,7 @@
         "testReimbursement",
         "licensingReimbursement",
         "tuitionTopUp",
-        "farmCoop"
+        "cooperativeTraining"
```

Comment:
Question for @lihanli or @markolson: Now that we're switching the options for this field from `farmCoop` to `cooperativeTraining`, what is someone is in the middle of completing this form while we deploy this change, had selected `farmCoop` and then submit the form, their submission won't pass the new schema, correct? How should we account for this?